spec.bs

```diff
@@ -1103,11 +1105,15 @@ To <dfn>report result</dfn> given a [=leading bid info=] |leadingBidInfo|:
   1. Otherwise, if |winner|'s [=generated bid/modified bid=] is not null, [=map/set=]
     |browserSignals|["`modifiedBid`"] to it.
   1. Set |config| to |leadingBidInfo|'s [=leading bid info/auction config=].
+  1. Set |sellerSignals| to |config|'s [=auction config/seller signals=].

```

Comment:
The _sellerSignals_ variable does not yet exist I believe. If this is meant to be its first-time declaration, we have to use "Let".